### PR TITLE
Add new OMERO features for 5.4

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/UloMh1fl/108-document-new-features-in-whats-new-pages-and-on-website

This adds restricted admins, rephrases reverse intensity rendering to reflect the name change and tweaks the subheader text to give us more leeway on when we update it

### CODE REVIEW

**Comment by hflynn** [comment]
@lunson I've re-used the manage users icon for now but do you have time to make us a new icon for restricted administrators?

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/new/

**Comment by lunson** [comment]
@hflynn Yup, can get this done today.

**Comment by hflynn** [comment]
@lunson thanks!

**Comment by lunson** [comment]
@hflynn Here you go: https://github.com/lunson/www.openmicroscopy.org/tree/add-new-omero-icon

**Comment by hflynn** [comment]
N.B. administrator guide link for restricted admins won't work until 5.4.0 docs are live but you can view the content if you substitute 5.4.0-m2 for the version number in the URL

**Comment by joshmoore** [comment]
New page looks good. :+1: I have issues with the target URL of the new section, but I'll bring that up on `ome-docs`.

File: omero/new/index.html [hunk 1]
```diff
@@ -10,11 +10,17 @@
...
-            The latest major release of OMERO includes the following new functionality
+            The most recent releases of OMERO include the following new functionality
         </p>
         <hr class="medium-10">
         <div class="row small-12 medium-up-3">
             <div class="omero-feature-container column text-center">
+                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_restricted-admins.svg">
+                <h4>Restricted Admins</h4>
+                <p>New configurable user role allowing trusted users to act on behalf of others.</p>
+                <a class="button hollow tiny" href="https://docs.openmicroscopy.org/latest/omero/sysadmins/admins-with-restricted-privileges.html" target="_blank">View administrator guide</a>
```
Comment (rleigh-codelibre) [inline omero/new/index.html]: This URL doesn't work.  Is that intentional?

Comment (mtbc) [reply omero/new/index.html]: Looks good as https://docs.openmicroscopy.org/omero/5.4.0-m3/sysadmins/admins-with-restricted-privileges.html.

File: omero/new/index.html [hunk 2]
```diff
@@ -27,21 +33,15 @@ <h2 class="header-blue">NEW Features</h2>
...
                 <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_plugin.svg">
                 <h4>Open With...</h4>
                 <p>Launch custom plugins via OMERO.web.</p>
                 <a class="button hollow tiny" href="http://help.openmicroscopy.org/web-client.html" target="_blank">View user guide</a>
             </div>
             <div class="omero-feature-container column text-center">
                 <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_reverse-intensity.svg">
-                <h4>Reverse intensity rendering</h4>
-                <p>Reverse or invert image intensity.</p>
+                <h4>Invert intensity rendering</h4>
```
Comment (rleigh-codelibre) [inline omero/new/index.html]: Inverted?

File: omero/new/index.html [hunk 3]
```diff
@@ -27,21 +33,15 @@ <h2 class="header-blue">NEW Features</h2>
...
+                <p>Invert the display of image colors according to intensity.</p>
```
Comment (rleigh-codelibre) [inline omero/new/index.html]: I find this phrasing a little confusing.  We're not "inverting the display", we are inverting the intensity of a channel for display.

File: omero/new/index.html (hunk 1 above)
Comment (hflynn) [reply omero/new/index.html]: Yep, as mentioned in a comment above, it'll work now with an milestone version number, the latest redirect will be live when we release.

**Comment by rleigh-codelibre** [comment]
Thanks.  Looks good to merge to me.

**Comment by hflynn** [comment]
Will need updating once https://github.com/openmicroscopy/ome-documentation/pull/1783 is merged.

**Comment by hflynn** [comment]
Update done now since docs PR looks like it should be going in. Also updated a few mentions of system admins to just admins for managing groups etc and improved links in a few other bits of the OMERO features/facility managers sections that I noticed.

**Comment by will-moore** [comment]
All looks good to me. Good to merge.

**Comment by hflynn** [comment]
Removing on hold label now 5.3.5 is out.

**Comment by hflynn** [comment]
Merging for 5.4.0 release

**Pull request merged**